examples/compute_instance/multiple_interfaces/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      version = ">= 3.45.0"
+    }
+    null = {
+      version = ">= 2.1.0"
+    }
+  }
+}
+
+# [START compute_vm_with_multi-if_create]
```

Comment:
I'm not sure if hyphens should be used in region tags? I might just call this `compute_vm_with_multiple_interfaces` unless this is matching an existing example.

Reply:
Done